sts/sts-gcp/src/main/java/com/salesforce/multicloudj/sts/gcp/GcpSts.java

```diff
@@ -63,15 +63,15 @@ protected StsCredentials getSTSCredentialsWithAssumeRole(AssumedRoleRequest requ
                 .setLifetime(Duration.newBuilder().setSeconds(request.getExpiration()).build())
```

Comment:
this seems to be the bug ? if no expiration is set, we shouldn't set the lifetime otherwise it will try to set zero as default which is not right.